uvcgan/cgan/README.md

```diff
@@ -0,0 +1,34 @@
+# Design Decisions
+
+When designing model architecture, I have made a number of decisions that
+seemed fine to me, but may cause problems in the future. I am going to
+summarize them below.
+
+
+## Domains
+
+I have hardcoded existence of only two domains --  domain `a` and domain `b`.
```

Comment:
@usert5432   This domain description likely need to be modified or removed.

Reply:
Sorry, missed that file. Fixed in https://github.com/LS4GAN/uvcgan/pull/1/commits/3795e23c95bcbd4cb05df051f62ceb8eb4c08de1. Thank you!